src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -68,12 +63,19 @@ export default class AgreementType extends Vue {
   @Prop({ default: false })
   private isSummary: boolean
 
+  // Global getters
+  @Getter getIncorporationAgreement!: Array<IncorporationAgreementTypeIF>
+
   // Actions
   @Action setIncorporationAgreementStepData!: ActionBindingIF
   @Action setIgnoreChanges!: ActionBindingIF
 
   private agreementType: string | null = null
 
+  private get agreementTypeDescription (): string {
+    return this.getIncorporationAgreement.find(x => x.code === this.agreementTypeState)?.description
```

Comment:
Can this `this.getIncorporationAgreement` have more than one true for this condition: `x.code === this.agreementTypeState` ?
If yes, this will get the first one in the array...

Reply:
Currently no, as the code is unique between the options in the `incorporationAgreement`. 
The `agreementTypeState` and `code` certainly can have the same value across the various resources but we only load in 1 resource based on the entity type, so there would only ever be the options present int the resource file. 
resource example: 
![Screen Shot 2021-05-31 at 1 32 13 PM](https://user-images.githubusercontent.com/53542131/120240240-ed38f700-c214-11eb-93d0-287a60ea836a.png)